modules/installation-osp-accessing-api-no-floating.adoc

```diff
@@ -3,10 +3,88 @@
 // * installing/installing_openstack/installing-openstack-installer.adoc
 // * installing/installing_openstack/installing-openstack-installer-custom.adoc
 // * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+// * installing/installing_openstack/installing-openstack-user.adoc
+// * installing/installing_openstack/installing-openstack-user-kuryr.adoc
+
+ifeval::["{context}" == "installing-openstack-installer-custom"]
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-kuryr"]
+:osp-kuryr:
+:osp-ipi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user"]
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-user-kuryr"]
+:osp-kuryr:
+:osp-upi:
+endif::[]
+ifeval::["{context}" == "installing-openstack-installer-restricted"]
+:osp-ipi:
+:osp-restricted:
+endif::[]
 
 [id="installation-osp-accessing-api-no-floating_{context}"]
-= Enabling access without floating IP addresses
+= Completing installation without floating IP addresses
+
+// FIP _and_ external network, or FIP _or_ external network?
+You can install {product-title} on {rh-openstack-first} without using a floating IP address and external network. In the
+ifdef::osp-ipi[`install-config.yaml`]
+ifdef::osp-upi[`inventory.yaml`]
+file, leave the following
+ifdef::osp-ipi[parameters]
+ifdef::osp-upi[variables]
+blank:
+
+ifdef::osp-ipi[]
+* `platform.openstack.ingressFloatingIP`
+* `platform.openstack.lbFloatingIP`
+* `platform.openstack.externalNetwork`
+endif::osp-ipi[]
+
+ifdef::osp-upi[]
+* `os_api_fip`
+* `os_ingress_fip`
+* `os_external_network`
+endif::osp-upi[]
+
+ifdef::osp-ipi[]
+If you run the installer from a system that cannot reach the {product-title} API due to a lack of floating IP addresses or name resolution, installation fails. To prevent installation failure in this case, you can use a proxy network or run the installer from a system that is on the same network as your {product-title} machines.
+endif::osp-ipi[]
+
+ifdef::osp-upi[]
+If you run the installer with the `wait-for` option from a system that cannot reach the {product-title} API due to a lack of floating IP addresses or name resolution, installation fails. To prevent installation failure in this case, you can use a proxy network or run the installer from a system that is on the same network as your {product-title} machines.
+endif::osp-upi[]
+
+[NOTE]
+====
+You can enable name resolution by creating DNS records for the API and Ingress ports. For example:  
++
```

Comment:
Extra `+`.
https://osp-fipless-osdocs1288--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-osp-accessing-api-no-floating_installing-openstack-installer-custom

Also, it would be nice to also include the Note about `/etc/hosts`, same as in the Enabling access with floating IP addresses section:
> If you do not control the DNS server, you can add the record to your /etc/hosts  file. This action makes the API accessible to you only, which is not  suitable for production deployment, but does allow installation for  development and testing.